packages/animations/README.md

```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
 
 ### Material's [Open Container Transitions](https://material.io/design/motion/choreography.html#transformation)
```

Comment:
Having the link in the title actually looks a little strange. Maybe we should have a dedicated link to the spec in the text below?

Reply:
We can do that but not yet, since the actual links are not available yet. I'll remove them for now